auto2cmake.py

```diff
@@ -1064,8 +1064,12 @@ def generate_default_cmake(req_dir):
     sources += ")\n"
 
     r_cmake_file = open(req_dir + "/CMakeLists.txt", "w")
-    r_cmake_file.write("cmake_minimum_required(VERSION 2.8)\n")
+    r_cmake_file.write("cmake_minimum_required(VERSION 3.24)\n")
```

Comment:
I know, that the original version didn't had it either, but what about specifying the minimum cmake version via a command line option?

Reply:
Sounds great. :-)